crates/wasi-common/wasi-misc-tests/src/bin/poll_oneoff.rs

```diff
@@ -98,6 +98,21 @@ unsafe fn test_stdin_read() {
         wasi_unstable::EVENTTYPE_CLOCK,
         "the event.type_ should equal clock"
     );
+
+    // Write to file
+    let contents = &['c' as u8];
+    let ciovec = wasi_unstable::CIoVec {
+        buf: contents.as_ptr() as *const libc::c_void,
+        buf_len: contents.len(),
+    };
+    let mut nwritten = 0;
+    let status = wasi_fd_write(wasi_unstable::STDIN_FD, &[ciovec], &mut nwritten);
```

Comment:
Shouldn't this be writing to `STDOUT_FD` instead of `STDIN_FD`? This seems to be causing the test failures in the CI BTW, resulting in `__WASI_ENOTCAPABLE` as we're missing `__WASI_RIGHT_FD_WRITE` right on the `STDIN_FD`.

Reply:
I intended to remove it. :)

Reply:
This is removed now.